### PR TITLE
Frontend configuration

### PR DESCRIPTION
**Pull request by MosesNwaeze**
## Description
- component directory where all the components are kept
- redux directory that manage states within the application
- styles directory where all the styles are kept
- All image are kept in the public/image directory. 
You are free to use bootstrap for your page layout, it has already being included for you.
## How Has This Been Tested?
I test the functionality of each module and the interaction between them and they all work fine. you can run the index page, you will see a demo of the interaction between the functionality of the modules

## Checklist:

- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x] I have added necessary inline code documentation
- [x] I have added tests that prove my fix is effective and that this feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by emydame — APPROVED** [review]
Commits meets expected standards. Greet job Moses

File: .prettierrc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "singleQuote": true,
+  "singleQuote": false,
```
Comment (Onyedikachi) [inline .prettierrc:2]: This change will affect the code quality going forward. We may be distorting Andela-Codacy config

File: src/backend/bin/www [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+server.listen(port);
+server.on("error", onError);
+server.on("listening", onListening);
+
+/**
+ * Normalize a port into a number, string, or false.
+ */
+
+function normalizePort(val) {
```
Comment (Onyedikachi) [inline src/backend/bin/www:41]: Let us stick with es6 arrow functions for consistency

**Review by Onyedikachi — APPROVED** [review]

File: .prettierrc (hunk 1 above)
Comment (MosesNwaeze) [reply .prettierrc:2]: i dont think so codacy demands all string to be double-quoted. it a must. most of the boiler plate code from andela failed codacy test

File: src/backend/bin/www (hunk 2 above)
Comment (MosesNwaeze) [reply src/backend/bin/www:41]: okay

**Review by fid37is — APPROVED** [review]

**Review by fid37is — APPROVED** [review]

**Pull request merged**